LevelUp.Pos.ProposedOrders.Tests/Mocks/PointOfSale.cs

```diff
@@ -1,11 +1,13 @@
-﻿namespace LevelUp.Pos.ProposedOrders.Tests.Mocks
+﻿using System;
+
+namespace LevelUp.Pos.ProposedOrders.Tests.Mocks
 {
     /// <summary>
     /// This class is meant to streamline expected calculations from a typical point of sale for the purpose of 
     /// testing.  The behavior of this class expects a starting total amount due (including tax), and the tax amount.
     /// It handles applying pre-tax discounts and standard payments/tenders.
     /// </summary>
-    internal class Check
+    internal class PointOfSale
```

Comment:
Maybe `PointOfSaleCheck` ?

Reply:
Are you saying that `PointOfSale` is confusing?  I'm not really sure what we gain by naming it `PointOfSaleCheck` (personally, that term is confusing).  I prefer `PointOfSale`.  It has an `.ApplyTender` method and an `.ApplyDiscount` method, among other "PointOfSale" characteristics.

Reply:
I found `PointOfSale` confusing, but at the same time this is mock test class and I don't have a strong feeling about it.